engines/registry/spec/app/models/access_scope_spec.rb

```diff
@@ -157,6 +157,7 @@
 
         it 'returns default auth actions (no free repos included)' do
           yaml_string = access_policy_content
+          FileUtils.cp('engines/registry/spec/data/access_policy_yaml.yml', 'engines/registry/spec/data/access_policy_yaml_old.yml')
```

Comment:
I suggest to pull out this unrelated change from this pr into another one, refactoring the registry access tests. I don't understand why the file is not at 'spec/data/access_policies.yml' and needs to get copied around.

Reply:
> I don't understand why the file is not at 'spec/data/access_policies.yml' and needs to get copied around

the files is `spec/data/access_policies_yaml.yml` which of course can be renamed, it gets _moved_ because
the tests checks the content, it gets to its original content so no `.gitignore` is needed